src/templates/functions.jsx

```diff
@@ -2,16 +2,96 @@ import React from 'react';
 
 import { html as toHtml } from '../indices-to-html';
 
+/**
+ * Adds markup to some common text patterns
+ *
+ *  - Bullet lists
+ *  - Todo lists, WP style
+ *  - Inline `code` snippets
+ *  - Header: description/explanation paragraphs
+ *
+ * Note: This code is only meant to serve until a
+ * proper parser can be built up to convert the
+ * unstructured text into structured data. Since at
+ * this time we still create HTML strings directly
+ * and on every render this function will serve
+ * sufficiently but it should not be looked upon
+ * as good example code!
+ *
+ * @param {String} text input list of blocks as HTML string
+ * @returns {String} marked-up text
+ */
+const toBlocks = text =>
+    text.split('\n').reduce(({ out, inList }, raw) => {
+        // emphasized definition-like text
+        // Some value: some description
+        // Header: Value
+        //
+        // Not! This is fun. This: again; isn't emphasized.
+        // May detect false positive if colon found in first sentence.
+        const defined = /^[\w\s-_]+:/.test(raw)
+            ? `<strong>${raw.split(':')[0]}:</strong>${raw.replace(/^[^:]:+/, '')}`
+            : raw;
+
+        // inline `code` snippets
+        const coded = defined.replace(/`([^`]+)`/, '<code>$1</code>');
```

Comment:
@gwwar I can't reproduce. are we sure that isn't actually a bullet followed by inline code? (inline code already has precedence over lists)

can you share the underlying text causing this issue?